nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -119,21 +119,22 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
             activations_checkpoint_method=cfg.get('activations_checkpoint_method', None),
             activations_checkpoint_num_layers=cfg.get('activations_checkpoint_num_layers', 1),
             layernorm_epsilon=cfg.get('layernorm_epsilon', 1e-5),
-            masked_softmax_fusion=cfg.get('masked_softmax_fusion', False),
-            bias_gelu_fusion=cfg.get('bias_gelu_fusion', False),
+            masked_softmax_fusion=cfg.get('masked_softmax_fusion', True),
+            bias_gelu_fusion=cfg.get('bias_gelu_fusion', True),
```

Comment:
Why are you changing the default values here?

Reply:
It was true before as default parameter via subsequent function calls
I had set to false while passing in as an explicit parameter, but to mimic the default function calls, I set it to True